### PR TITLE
[4.0] [a11y] PR for #29909 rfc empty headings

### PR DESCRIPTION
**Pull request by hans2103**
Pull Request for Issue # .

### Summary of Changes

This PR adds a fallback value for heading when creating a new user. 
Created for https://github.com/joomla/joomla-cms/pull/29909

### Testing Instructions

- Start creating a new user. 
- View source and find the first heading in id=user-form

### Actual result BEFORE applying this Pull Request

Before ... heading is empty

Schermafdruk 2020-07-03 01 04 15

### Expected result AFTER applying this Pull Request

After... heading is filled with text. Text is visible too,

Schermafdruk 2020-07-03 01 03 23

### CODE REVIEW

**Comment by brianteeman** [comment]
You do realise it's not just for com_users ?

**Comment by hans2103** [comment]
@brianteeman small steps.

**Comment by hans2103** [comment]
if this idea is accepted, I will continue with the rest.

**Comment by brianteeman** [comment]
Was just checking. 

To match the style guide each word in the title string should be capitalized.

Calling it a night will check again in the morning

**Comment by ceford** [comment]
I have tested this item :white_check_mark: successfully on b03f92cd03848479065653e2647b6ec8b04bc08f<br /><br />The instructions might have said to look for id="user-form" and the empty h2 is in the next line. I guess the capitalisation is a just a minor string fix. For a user edit the h2 contains the user name which might be John Doe or just john.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29920">issues.joomla.org/tracker/joomla-cms/29920</a>.</sub>

**Comment by hans2103** [comment]
> To match the style guide each word in the title string should be capitalized.

@brianteeman can you point me where this is not the case?

File: administrator/language/en-GB/com_users.ini [hunk 1]
```diff
@@ -316,6 +316,7 @@ COM_USERS_USER_FIELD_TWOFACTOR_LABEL="Authentication Method"
 COM_USERS_USER_FIELD_USERNAME_LABEL="Login Name (Username)"
 COM_USERS_USER_GROUPS_HAVING_ACCESS="User Groups With Viewing Access"
 COM_USERS_USER_HEADING="User"
+COM_USERS_USER_NEW_USER_TITLE="Create new user"
```
Comment (brianteeman) [inline administrator/language/en-GB/com_users.ini]:
```suggestion
COM_USERS_USER_NEW_USER_TITLE="Create New User"
```

Comment (hans2103) [reply administrator/language/en-GB/com_users.ini]: thank you... found it and fixed it.

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on e15524d3f48737258b7880072d590ff3290ed824<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29920">issues.joomla.org/tracker/joomla-cms/29920</a>.</sub>

**Comment by brianteeman** [comment]
@hans2103 as an RFC I am marking this as success. Thanks.

Thinking afresh on the wording I am thinking it would be better to have a descriptive text instead of an action text. In other words "New User Details" instead of "Create....<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29920">issues.joomla.org/tracker/joomla-cms/29920</a>.</sub>

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 2e858c35deb192938f5bfe967bf7b59988a9f103<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29920">issues.joomla.org/tracker/joomla-cms/29920</a>.</sub>

**Comment by hans2103** [comment]
> You do realise it's not just for com_users ?

I'm failing to find other components with this behaviour. The edit.php file from other components work differently.


```
<?php echo LayoutHelper::render('joomla.edit.title_alias', $this); ?>
```

**Comment by brianteeman** [comment]
administrator/index.php?option=com_languages&view=language&layout=edit

**Comment by brianteeman** [comment]
I'm sure I found more than that before but can't see any right now

**Comment by alikon** [comment]
please open a new issue for that..... in the meantime let me close this one

**Comment by alikon** [comment]
oops wrong one

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 2e858c35deb192938f5bfe967bf7b59988a9f103<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29920">issues.joomla.org/tracker/joomla-cms/29920</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29920">issues.joomla.org/tracker/joomla-cms/29920</a>.</sub>

**Pull request merged**